### ISSUE TITLE
CSS font-size not working

### ISSUE THREAD

**Issue by Dr-BW** [plutext/docx4j#384]
HTML with a CSS font-size defined in pts (and others) does not work.
For example:
```
WordprocessingMLPackage wordMLPackage = WordprocessingMLPackage.createPackage()
XHTMLImporterImpl importer = new XHTMLImporterImpl(wordMLPackage)
importer.setParagraphFormatting(FormattingOption.CLASS_PLUS_OTHER)
importer.setRunFormatting(FormattingOption.CLASS_PLUS_OTHER)
String html = "<html><head><style>body { font-family: helvetica, sans-serif; font-size: 24pt; color:#FF00F0; }</style></head><body>Hello World!</body></html>"
importer.convert(html, null)
```
I believe the issue is in [org.docx4j.model.properties.run.FontSize.java](https://github.com/plutext/docx4j/blob/master/docx4j-core/src/main/java/org/docx4j/model/properties/run/FontSize.java#L110) - lines 110, 114. The if-test clauses are "not equals" for CSS_EMS and CSS_EXS. This means that anything not CSS_EMS will always hit the CSS_EMS code block at that point. Presumably the if-test should be "equals".

**Comment by hydra1983** [comment]
+1

**Comment by mksharma17** [comment]
+1

**Comment by mksharma17** [comment]
it was working fine in docx4j.jar but not working in docx4j-core.jar

**Comment by mksharma17** [comment]
When it will be fixed?

**Comment by mandeepksharma99** [comment]
+1

**Comment by plutext** [comment]
https://github.com/plutext/docx4j/commit/68a9ecf9a73e6fc8fef56b0939dcefc1eb1f9809

**Comment by plutext** [comment]
Fix is in 8.2.6
